Makefile

```diff
@@ -21,6 +21,7 @@ ES_OPERATOR_NAMESPACE ?= openshift-logging
 ES_OPERATOR_BRANCH ?= release-4.3
 PROMETHEUS_OPERATOR_TAG ?= v0.34.0
 ES_OPERATOR_IMAGE ?= quay.io/openshift/origin-elasticsearch-operator:4.3
+ES_PROXY_IMAGE ?= pavolloffay/elasticsearch-proxy:latest
```

Comment:
TODO: change this once the image is published

Reply:
This does not seem to be needed at the moment as the ES operator has correct proxy (`latest`) and the tag for `4.4` does not exist https://quay.io/repository/openshift/origin-elasticsearch-proxy?tab=tags

```
    Environment:
      WATCH_NAMESPACE:      
      OPERATOR_NAME:        elasticsearch-operator
      PROXY_IMAGE:          quay.io/openshift/origin-oauth-proxy:v4.0.0
      ELASTICSEARCH_PROXY:  quay.io/openshift/origin-elasticsearch-proxy:latest
      ELASTICSEARCH_IMAGE:  quay.io/openshift/origin-logging-elasticsearch6:latest
```

Reply:
I will change the value to `quay.io/openshift/origin-elasticsearch-proxy:latest` so we don't forget it when upgrading to 4.5